complex_import/manual.md

```diff
@@ -163,14 +184,10 @@ As soon as the glTF file changed, you can immediately observe the new geometry i
 ![](./docs/voxel_modifier_5.png)
 
 You can arbitrarily modify all meshes' geometry and re-export - and the mesh's changes will be automatically imported by the Composer.
-There is a limitation though: the nodes and their transformations will not be re-imported. If you move an object or rotate it differently - this change
-will not be re-imported by the Composer. As a workaround, you can import the entire file again, copy-paste only the nodes that you want re-imported, then delete
+Note that nodes and their transformations will not be re-imported. If you move an object or rotate it differently - this change
+will not be re-imported by the Composer. You can import the entire file again, copy-paste only the nodes that you want re-imported, then delete
 the second copy of the import.
 
```

Comment:
With the new asset import dialog, the user does not need to reimport "the entire file" anymore - worth mentioning & changing the paragraph here?